### ISSUE TITLE
Marketing through content writing

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#52]
# TL;DR
Build a follower base, a reputation, for content, product and service

# Key Takeaways

- Marketing isn’t what goes viral. Because marketing isn’t interested in delivering real value to a reader or a viewer. It’s there to poke and prod you along until you perform the desired result.

- They want to experience your content the same way they would experience grabbing a cup of coffee with you at a local coffee shop.

- The entrepreneurs that understand how to communicate as people are the ones who build audiences and ultimately win.

- If you’re an entrepreneur, of any kind, you need to realize the single greatest investment you will ever make in yourself is in telling your personal story through the content you share.

# Link
“The Subtle Difference Between Good And Great Marketers” by Nicolas Cole https://link.medium.com/0M9CraalH5
